ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -88,74 +98,98 @@ const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
-      </React.Fragment>
-    );
-  }
-
-  public render(): React.ReactNode {
-    const selectedWallet = this.props.navigation.getParam("selectedWallet");
-
-    const isFavorite = pot.map(
-      this.props.favoriteWallet,
-      _ => _ === selectedWallet.idWallet
-    );
-
-    // to retro-compatibility purpose
-    const pm = pot.getOrElse(
-      pot.map(this.props.paymentMethods, pms =>
-        pms.find(pm => pm.idWallet === selectedWallet.idWallet)
-      ),
-      undefined
-    );
-
-    return (
-      <WalletLayout
-        title={I18n.t("wallet.paymentMethod")}
-        allowGoBack={true}
-        topContent={this.headerContent(selectedWallet, isFavorite)}
-        hideHeader={true}
-        hasDynamicSubHeader={true}
-        topContentHeight={HEADER_HEIGHT}
-        contextualHelpMarkdown={contextualHelpMarkdown}
-        faqCategories={["wallet_transaction"]}
-      >
-        {pm && (
-          <>
-            <View style={IOStyles.horizontalContentPadding}>
-              <View spacer={true} extralarge={true} />
-              <PaymentMethodCapabilities paymentMethod={pm} />
-              <View spacer={true} />
-              <ItemSeparatorComponent noPadded={true} />
-            </View>
-            <EdgeBorderComponent />
-          </>
-        )}
-      </WalletLayout>
-    );
-  }
-}
+const headerContent = (
+  selectedWallet: Wallet,
+  isFavorite: pot.Pot<boolean, Error>,
+  setFavorite: (walletId?: number) => void,
+  onDelete: (walletId: number) => void
+) => (
+  <>
+    <View style={styles.cardBox}>
+      <CardComponent
+        type={"Header"}
+        wallet={selectedWallet}
+        hideFavoriteIcon={false}
+        hideMenu={false}
+        isFavorite={isFavorite}
+        onSetFavorite={(willBeFavorite: boolean) =>
+          handleSetFavourite(willBeFavorite, () =>
+            setFavorite(selectedWallet.idWallet)
+          )
+        }
+        onDelete={() => onDelete(selectedWallet.idWallet)}
+      />
+    </View>
+  </>
+);
+
+const TransactionsScreen: React.FC<Props> = (props: Props) => {
+  const selectedWallet = props.navigation.getParam("selectedWallet");
+
+  const isFavorite = pot.map(
+    props.favoriteWallet,
+    _ => _ === selectedWallet.idWallet
+  );
+
+  // to retro-compatibility purpose
+  const pm = pot.getOrElse(
+    pot.map(props.paymentMethods, pms =>
+      pms.find(pm => pm.idWallet === selectedWallet.idWallet)
+    ),
+    undefined
+  );
+  const { present } = useRemovePaymentMethodBottomSheet({
+    icon: selectedWallet.creditCard
+      ? getCardIconFromBrandLogo(selectedWallet.creditCard)
+      : defaultCardIcon,
+    caption:
+      selectedWallet.paymentMethod &&
+      isRawCreditCard(selectedWallet.paymentMethod)
+        ? getTitleFromCard(selectedWallet.paymentMethod)
+        : FOUR_UNICODE_CIRCLES
+  });
+
+  const UnsubscribeButton = (props: { onPress?: () => void }) => (
+    <Button bordered={true} style={styles.cancelButton} onPress={props.onPress}>
```

Comment:
```suggestion
    <ButtonDefaultOpacity bordered={true} style={styles.cancelButton} onPress={props.onPress}>
```
This is the default button looks&feel (no feedback on onpress)

Reply:
@Undermaken the bottom sheet *is* the alert to confirm the destructive action. If you see any potential harm in doing so (e.g.: bottom sheet can be dismissed by unknown custom behaviors by vendor OS), then let's only use the standard alert. But having both is redundant.

If we keep the alert, "ok" CTA needs to be "Elimina / Delete" (ux principle: actions should always be self-explanatory)
If we keep the bottom sheet, the delete CTA has to be red, as we did in "Cancella iscrizione dal Cashback"

Reply:
Sound good!
I didn't figure out the BS is a confirmation step.
From my POV this should be considered resolved ;)

Reply:
I modified the button's color inside the bottomsheet in [d3a32da](https://github.com/pagopa/io-app/pull/2917/commits/d3a32daef2cb177a7ff57a46dd4614ab249aae6e).
Notice that this update the color of all buttons in the bottomsheets of the same type ( BancomatDetailScreen, BPayDetailScreen, etc..) 

<img src="https://user-images.githubusercontent.com/11773070/111775201-bb202600-88b0-11eb-95b4-0caa262a512e.png" width=300>